docs/07-git-cheat-sheet.md

```diff
@@ -1 +1,22 @@
-TEST
\ No newline at end of file
+# Creating a pull request from master
+Step# | Description | Command
+------------ | ------------- | -------------
+1. | Create a new brach (If you need one) | git cb branch-name
```

Comment:
Should probably change it to the list format we just found.

1.
1.
1.

Reply:
Unfortunately the number formatting doesn't work in the table